content/general/concepts/github/github.md

```diff
@@ -0,0 +1,17 @@
+---
+Title: 'Github'
+Description: 'Github is a UI based version control that uses git.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'GitHub'
+  - 'Git'
+CatalogContent:
+  - "learn-git"
+  - "paths/learn-how-to-build-websites"
```

Comment:
```suggestion
  - 'learn-git'
  - 'paths/front-end-engineer'
```

Single-quotes for meta-data and let's go with `paths/front-end-engineer` since it has a specific section about Git and GitHub.

Reply:
@Dusch4593 I think we should keep it as paths/learn-how-to-build-websites since a larger % of the path is on github